crates/paimon/Cargo.toml

```
@@ -16,14 +16,15 @@
 # under the License.
 
 [package]
-categories = ["command-line-utilities"]
-description = "The rust implementation of paimon"
+categories = ["database"]
+description = "The rust implementation of Apache Paimon"
```

```suggestion
description = "The Rust implementation of Apache Paimon"
```
Do initial letters need to be capitalized?

---

Both look to me.